Cargo.toml

```diff
@@ -54,6 +54,7 @@ simplelog = "0.12.0"
 kanata-keyberon = { path = "keyberon" }
 kanata-parser =   { path = "parser" }
 kanata-tcp-protocol = { path = "tcp_protocol" }
+indoc = "2.0.4"
```

Comment:
Let's move this up to the other non-local dependencies under `dirs`.

Also, seems like this could be optional, so I think it is better to have it as such. See how the `interception-driver` dependency and `interception_driver` feature work.